packages/pulp/python-yarl/python-yarl.spec

```diff
@@ -0,0 +1,47 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name yarl
+
+Name:           python-%{pypi_name}
+Version:        1.3.0
+Release:        1%{?dist}
+Summary:        Yet another URL library
+
+License:        Apache 2
+URL:            https://github.com/aio-libs/yarl/
+Source0:        https://files.pythonhosted.org/packages/source/y/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+
+%description
+ :target: :target:
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+
+Requires:       python3-idna >= 2.0
+Requires:       python3-multidict >= 4.0
+%description -n python3-%{pypi_name}
+ :target: :target:
```

Comment:
This seems curious.

Reply:
it's what's in their README.rst (or rather, how it's mangled on pyp2rpm) and I don't feel like fixing that given the packages are not for general use outside of being a dependency for pulpcore.